### PR TITLE
Feat/cloud terminal api

### PR DESCRIPTION
**Pull request by gcatanese**
Implement In-Person Payments use case

Note: README file has been also updated

### CODE REVIEW

File: in-person-payments-example/public/images/.keep
Comment (Kwok-he-Chu) [inline in-person-payments-example/public/images/.keep:1]: Empty file

**Review by Kwok-he-Chu — CHANGES_REQUESTED** [review]
Good work splitting the endpoint routes into the separate files

- [ ] Update gitpod file so that it takes the in-person-payments-example into account.
- [ ] When terminal is turned off, and trying to make a payment request, the application crashes "paymentResponse is undefined"

File: in-person-payments-example/public/images/.keep
Comment (gcatanese) [reply in-person-payments-example/public/images/.keep:1]: Removed

File: in-person-payments-example/routes/webhookRoute [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        saveTable(table);
+
+    } else if (notification.eventCode == "REFUND_FAILED") {
+        // webhook with payment refund failure
+        console.log("Payment refund failed - pspReference:" + notification.pspReference + " eventCode:" + notification.eventCode);
+
+        const saleTransactionId = notification.merchantReference;
+        const table = getTableBySaleTransactionId(saleTransactionId);
+
+        table.paymentStatus = global.STATUS_REFUNDFAILED;
```
Comment (Kwok-he-Chu) [inline in-person-payments-example/routes/webhookRoute:75]: Optional: it's good practice to check the "succes"-flag here as well

File: in-person-payments-example/routes/webhookRoute [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        saveTable(table);
+
+    } else if (notification.eventCode == "REFUNDED_REVERSED") {
+        // webhook with payment refund reversed
+        console.log("Payment refund reversed - pspReference:" + notification.pspReference + " eventCode:" + notification.eventCode);
+
+        const saleTransactionId = notification.merchantReference;
+        const table = getTableBySaleTransactionId(saleTransactionId);
+
+        table.paymentStatus = global.STATUS_REFUNDREVERSED;
```
Comment (Kwok-he-Chu) [inline in-person-payments-example/routes/webhookRoute:85]: Optional: it's good practice to check the "succes"-flag here as well

**Review by Kwok-he-Chu — APPROVED** [review]
Good job! Node IPP done!

**Pull request merged**